docs/README.md

```diff
@@ -5,46 +5,57 @@ HyP3 documentation is build using [MkDocs](https://www.mkdocs.org/) and the
 
 ## How to
 
-### setting up a development environment
+### Setting up a development environment
 
-From the repository root, you can setup a conda environment with all the 
-necessary dependencies
+In order to automatically document some of our APIs, we use a `conda` environment
+with our APIs installed. You can get Miniconda (recommended) here:
+
+https://docs.conda.io/en/latest/miniconda.html
+
+Once conda is installed, from the repository root, you can create and activate a 
+conda environment with all the necessary dependencies
 
 ```
 conda env create -f conda-env.yml
+conda activate ASFHyP3
 ```
 
-### build and view the documentation site
+Later, you can update the environments dependencies with
+
+```
+conda env update -f conda-env.yml
+```
+
+### Build and view the documentation site
+
+With the `ASFHyP3` conda environment activated, run
+
 ```
 mkdocs serve
 ```
 
-which will allow you to view the documentation at http://127.0.0.1:8000/. This
-make target will automatically watch for new/changed files in this directory and
+to generate the documentation. This will allow you to view it at http://127.0.0.1:8000/.
+MkDocs will automatically watch for new/changed files in this directory and
 rebuild the website so you can see your changes live (just refresh the webpage!).
 
 *Note: Because this captures your terminal (`crtl+c` to exit), it's recommended you
```

Comment:
Maybe this will make sense to people who do this more, but the ctrl+c seems out of place to me. Are you suggesting that people exit their terminal? And open a fresh one so that it doesn't include stuff that you did before that you don't want to include? If I'm understanding this right, perhaps something like this (except with the words that would be more usual):
*Note: Because this process captures the contents of your terminal, we recommend that you close your current terminal (ctrl+c) and run the mkdocs serve command in a fresh terminal.

Reply:
Ah no, when a process captures you're terminal, you can use `crtl+c` to kill the process, giving you back your terminal. So, using a second/separate/dedicated terminal allows you to keep `mkdocs serve` running

Reply:
Might be better to drop the "(`crtl+c` to exit)"; it's not really needed